### PR TITLE
add indexer api interaction instructions

### PR DESCRIPTION
**Pull request by MatejVukosav**
Add details on how to use indexer API.

Page:
Screenshot 2022-11-29 at 00 21 18

Screenshot 2022-11-29 at 00 21 24

### CODE REVIEW

File: docs/interact/indexer.mdx [hunk 1]
```diff
@@ -1,36 +1,76 @@
...
+### Configuration
 
-Create a new shard token
-https://near.github.io/near-api-js/interfaces/connect.connectconfig.html
+To be able to call Indexer API you must set few parameters:
+1. Endpoint
+2. Authorization token
 
-Generate a new token in the calimero security dashboard.
+#Endpoint
```
Comment (xilosada) [inline docs/interact/indexer.mdx:15]:
```suggestion
# Endpoint
```

File: docs/interact/indexer.mdx [hunk 2]
```diff
@@ -1,36 +1,76 @@
...
+Endpoint is constructed from multiple parts:
```
Comment (xilosada) [inline docs/interact/indexer.mdx:16]: This can be copied from the dashboard, we should mention it

**Review by xilosada — APPROVED** [review]
Check comments

File: docs/interact/indexer.mdx (hunk 2 above)
Comment (MatejVukosav) [reply docs/interact/indexer.mdx:16]: Added link to a subpage where it is explained. 
Screenshot 2022-11-29 at 00 29 37

Comment (MatejVukosav) [reply docs/interact/indexer.mdx:16]: Screenshot 2022-11-29 at 00 30 37

added link to the subpage where it is explained

Comment (MatejVukosav) [reply docs/interact/indexer.mdx:16]: Although looking at this section, seems like links are outdated. I will update them

**Pull request merged**